add-ons/che/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is it required?

Reply:
This could be a general statement... But it is right to discuss, as now the addons are public domain?

Reply:
I usually add it as practice, if you guys done require it will remove it

Reply:
@kameshsampath Yes, we have it at project level https://github.com/minishift/minishift-addons/blob/master/LICENSE

Reply:
Unnecessary to state again

Reply:
@kameshsampath We have the license for the project so you do not need another copy of the same license.